aiidalab_widgets_base/viewers.py

```diff
@@ -213,7 +213,8 @@ def __init__(
         # Constructing configuration box
         if configuration_tabs is None:
             configuration_tabs = ["Selection", "Appearance", "Cell", "Download"]
-
+        if len(configuration_tabs) != 0:
+            self.selection_tab_idx = 0
```

Comment:
I assume 0 is anyways the default, so we might just completely remove the line:
```suggestion
```

Reply:
That's not a correct assumption, `configuration_tab` is a constructor parameter so the Selection Tab index can be whatever, the selection tab might not even be there. 

That said, you've removed the `self.selection_tab_idx` completely in your previous PR, so this line should be removed anyway.